packages/css/src/components/tabs/tabs.scss

```diff
@@ -62,6 +62,10 @@
       background-color: SelectedItem;
     }
   }
+
+  * {
+    pointer-events: none;
+  }
```

Comment:
I think we're solving a problem that has already been solved here. If you need to access the element that has an `onClick` handler, you can use `event.currentTarget`. If you need to access the element that has been clicked, you can use `event.target`.

Adding CSS for that is unnecessary imo, and it breaks the second case.

Reply:
Oh my, good point. Reverted it and changed the PR title.